extensions/rates/functions/package.json

```diff
@@ -22,15 +22,13 @@
     "firebase": "^9.0.2",
     "firebase-admin": "^9.8.0",
     "firebase-functions": "^3.14.1",
-    "humps": "^2.0.1",
     "mocha": "^9.1.1",
     "shipengine": "^1.0.0",
     "shipengine-firebase-common": "^1.0.0",
     "typescript": "^3.8.0"
   },
   "devDependencies": {
     "@types/chai": "^4.2.21",
-    "@types/humps": "^2.0.1",
```

Comment:
There's no need to do the camelcaseKeys anymore, since we've moved to the castParams approach.